ChangeCalculation/ChangeCalculator.cs

```diff
@@ -1 +1,33 @@
-﻿
\ No newline at end of file
+﻿namespace ChangeCalculation
+{
+    public static class ChangeCalculator
+    {
+        private static readonly List<int> _validCoins = new List<int> { 100, 50, 10, 5, 2, 1 };
+
+        public static object CalculateChange(int change, List<int> availableCoins)
```

Comment:
Let's make the following improvements to the contract:

- rename change variable to amount, you ask for amount, not for change
- change the contract from not specific object to something like this (it is in pseudocode):
{
  CanReturn: boolean,
  Coins: List<int>
}
If cannot return, then Coins contains an empty list.